docs/solution-design.md

```diff
@@ -6,6 +6,50 @@ While there are many ways to achieve the goal we are looking for, for the purpos
 
 ## Components used and their roles
 
-## Message Flow
+The following components will be used as part of the solution:
+  
+- Cosmos DB Account
+- Cosmos DB Containers
```

Comment:
These containers are in a "inventory" database

```
Cosmos DB Account
    Cosmos DB Database (inventory)
        Cosmos DB Containers
           - On Hand (onHand)
           - Shipments (shipments)
           - Master Data Container (mdc)
```

Reply:
Got it. I will make the change.

Reply:
Done